File: dayFour/python/two_sum.py
def targeting_sum(arr, target):
    result = []
    for index in range(0, len(arr)):
        for secondIndex in range(1, len(arr)):
            if arr[index] + arr[secondIndex] == target:
                result.append(arr[index])
                result.append(arr[secondIndex])
                return result 
    return result

File: dayFour/javaScript/twoSum.js
export const targettingSum = (arr, target) {
    
    result = [0, 0];

    for (let index = 0; index < arr.length; index++) {
        
        for (let secondIndex = index + 1; secondIndex < arr.length; secondIndex++) {
          
            if (arr[index] + arr[secondIndex] == target) {
                result[0] = arr[index];
                result[1] = arr[secondIndex];
            }
        }
    }
    return result;
}


